CairoMakie/test/runtests.jl

```diff
@@ -134,6 +134,12 @@ end
     rm("test.mp4")
 end
 
+@testset "plotlist no ambiguity" begin
+    plotlist([Makie.SpecApi.Scatter(1:10)])
+    save("test.png", f)
```

Comment:
This could also be `Makie.colorbuffer(f; backend=CairoMakie)` to keep everything in memory